frontend/src/components/ViewTrips/save-trip-modal.js

```diff
@@ -140,10 +154,11 @@ class SaveTripModal extends React.Component {
       [DB.TRIPS_DESTINATION]: this.destinationRef.current.value,
       [DB.TRIPS_START_DATE]: this.startDateRef.current.value,
       [DB.TRIPS_END_DATE]: this.endDateRef.current.value,
-      [DB.TRIPS_COLLABORATORS]:
+      [DB.RAW_COLLAB_EMAILS]:
           this.state.collaboratorsRefArr.map(ref => ref.current.value),
     };
-    const tripData = await formatTripData(rawTripData);
+    const tripData = await TripUtils.formatTripData(rawTripData,
+                                                    this.props.tripData);
 
     if (this.isAddTripForm) {
```

Comment:
(I know this isn't part of this PR but) Where do you declare this?

Reply:
Declare what? 

- `formatTripData` is declared in `trip-utils.js`
- `rawTripData` is declared above

let me know if you were referring to something else.

Reply:
Yeah, this.isAddTripForm. Thanks!